dev/com.ibm.ws.microprofile.health.2.0/resources/com/ibm/ws/microprofile/health20/resources/Health20.nlsprops

```diff
@@ -50,6 +50,6 @@ healthcheck.application.down.CWMH0052W.explanation=The DOWN status might cause a
 healthcheck.application.down.CWMH0052W.useraction=No user action is required.  This warning is a record of the DOWN status.
 
 # The Readiness HealthCheck had a DOWN outcome, when applications are not started yet
-readiness.healthcheck.applications.not.started.down.CWMH0053W=CWMH0053W: The readiness health check reported a DOWN overall status, because the following applications has not started yet: {0}
+readiness.healthcheck.applications.not.started.down.CWMH0053W=CWMH0053W: The readiness health check reported a DOWN overall status because the following applications have not started yet: {0}
```

Comment:
Change
"The readiness health check reported a DOWN overall status because the following applications have not started yet: {0}"
to
"The readiness health check reported a DOWN overall status because the following applications are not started: {0}"